doc/overview/consensus/bft.rst

```diff
@@ -0,0 +1,167 @@
+Byzantine Fault Tolerance
+=========================
+
+Below we discuss the reconfiguration in a BFT setting. 
```

Comment:
Technically the 2-tx scheme is not specific to BFT, it could be used in CFT as well. We want to use it for BFT because:

1. it only starts when the reconfiguration transaction is committed, so a reconfiguration attempt can never roll back
2. because it is not atomic, it has room for additional conditions, such as checking that the byzantine reconfiguration (a multiple transaction protocol) is complete before proceeding to the new config.

Reply:
I included your comment in the text now 👍